build.gradle

```
@@ -1,6 +1,7 @@
 plugins {
     id "java"
     id "edu.wpi.first.GradleRIO" version "2024.1.1"
+    id("com.diffplug.spotless") version "6.23.3"
 }
```

It's good practice to keep the same style within the same file.

```suggestion
    id "com.diffplug.spotless" version "6.23.3"
```